cmd/stackdriver-prometheus-sidecar/main.go

```diff
@@ -228,6 +230,8 @@ func main() {
 	a.Flag("prometheus.api-address", "Address to listen on for UI, API, and telemetry.").
 		Default("http://127.0.0.1:9090/").URLVar(&cfg.prometheusURL)
 
+	a.Flag("stackdriver.enable-internal-metrics", "Whether to report Sidecar metrics to Stackdriver").Default("false").BoolVar(&cfg.sdOpencensus)
```

Comment:
I don't want to be too picky about this, but the flag name is unclear. I'd call it something like monitoring.backend and make the default value "prometheus" and have "stackdriver" as optional. WDYT?

Reply:
Thanks, that's a good idea. Renamed to `--monitoring.backend`.